packages/experimental-package/src/components/PageHeader/_storybook-styles.scss

```diff
@@ -11,34 +11,26 @@
 @import 'carbon-components/scss/components/tabs/tabs';
 @import 'carbon-components/scss/components/tag/tag';
 
-.page-header-stories__status-icon {
-  vertical-align: middle;
-  fill: $support-02;
-}
-
-.container {
-  height: 100vh;
-}
-
-.content {
-  width: 0;
-  height: 0;
-  margin-left: 5vw;
-  border-right: 45vw solid transparent;
-  border-bottom: 150vh solid $support-03;
-  border-left: 45vw solid transparent;
+.page-header-stories__viewport {
+  max-height: 500px;
```

Comment:
Why not just add a few more dummy blocks or block height? Looks a bit odd scrolling just part of the screen.

Broke PageHeaderCopy.stories.js

Reply:
i have reinstated `.content `in the styles.